PWGPP/TPC/macros/AddTaskTPCPIDEtaQA.C

```diff
@@ -86,21 +82,23 @@ AliAnalysisTask *AddTaskTPCPIDEtaQA(TString period = "", Bool_t isPbpOrpPb = kFA
   //below the trunk version
   AliAnalysisDataContainer *cinput  = mgr->GetCommonInputContainer();
 
-  //dumm output container
-  AliAnalysisDataContainer *coutput0 =
-      mgr->CreateContainer("TPCPIDEtaQA_tree",
-                           TTree::Class(),
-                           AliAnalysisManager::kExchangeContainer,
-                           "TPCPIDEtaQA_default");
-
   //define output containers, please use 'username'_'somename'
   AliAnalysisDataContainer *coutput1 = 
       mgr->CreateContainer("TPCPIDEtaQA", TObjArray::Class(),
                            AliAnalysisManager::kOutputContainer,"TPCPIDEtaQA.root");
 
   //connect containers
   mgr->ConnectInput  (task,  0, cinput );
-  mgr->ConnectOutput (task,  0, coutput0);
+  
+  if (mgr->GetCommonOutputContainer()) {
```

Comment:
Why is this protection necessary?

Reply:
Local trains dont have a CommonOutputContainer, so this fails in testing it locally.